lib/bulk_reindexer.rb

```diff
@@ -0,0 +1,15 @@
+module Argo
+  class BulkReindexer
+    def self.queue_pid_reindexing_jobs(pid_list, should_commit = false, should_profile = false, priority = 1, batch_size = 10)
+      pid_list.each_slice(batch_size) do |sublist|
+        IndexingJob.delay(priority: priority).perform_later(sublist, should_commit, should_profile)
+      end
+    end
+
+    def self.reindex_all(args, should_commit = true, should_profile = true)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used.